nss-day-cohort-41/nutshell-sea-rose
# Feature Login Checks

Features:
* Rearranged email to be the main login.
* Email and Password check for login
* Password Check for matching passwords on registration
* Email existence check

Testing Instruction:
* When the user puts in their email and password into the Login, and clicks Sign In the inputs are checked to make sure the email and password are correct in the database.
* When using Create Email when the user inputs the passwords for both fields. It will check to make sure that they are the same. If not then they will be alerted to the passwords not being identical.
* The email check happens when the user is creating a new account through registration. When registering the user will have their email checked to see if the email already exist in the database. If it already exist the user will be alerted of the existence of the other accounts existence.
